source/plugins/logging.rst

```diff
@@ -2,14 +2,73 @@
 ==========================
 
-Java comes with a few logging frameworks. Logging is preferrable to printing to stdout (or stderr) with ``System.out.println()`` for a number of reasons:
+There are a few logging frameworks available for use in Java. Logging is preferable to printing to stdout or stderr with ``System.out.println()`` for a number of reasons:
 
-* Logged messages have a source name attached to them so it is possible to figure out where the logged messages are coming from.
-* Logged messages have a severity level which allows for simple filtering (i.e. disable all non-critical notices).
+* Logged messages are labeled with a source name, making it easier to figure out where the logged messages are coming from.
+* Logged messages have a severity level which allows for simple filtering (e.g. disable all non-critical notices).
 * The available logger frameworks allow you to enable or disable messages from certain sources.
 
 Getting a Logger
 ================
 
+The Guice module used during the initialization of plugins has a plugin-scoped logger. This allows you to annotate a Logger field or object with ``@Inject`` to get the logger for your plugin, which is pre-configured with the correct plugin ID.
+
+.. note::
+    See :doc:`quick-start` for information on configuring your plugin ID.
+
+**Example - Field**
+
+.. code-block:: java
+
+    import com.google.inject.Inject;
+    import java.util.logging.Logger;
+
+    @Inject
+    private Logger logger;
+
+**Example - Method**
+
+.. code-block:: java
+
+    import com.google.inject.Inject;
+    import java.util.logging.Logger;
+
+    @Inject
+    private void setLogger(Logger logger)
+        this.logger = logger;
+    }
+
+It is recommended to set your logger in your main plugin class, as it is instantiated with the Guice injector when the plugin is loaded.
+
+Creating a getter method for your logger in the same class in which it was set is also ideal. An example getter method is illustrated below.
+
+.. code-block:: java
+
+    import com.google.inject.Inject;
+    import java.util.logging.Logger;
+
+    public Logger getLogger() {
+        return logger;
+    }
+
 Emitting Messages
-=================
\ No newline at end of file
+=================
+
+Emitting a message with your logger is very simple.
+
+.. note::
+
+    The following two examples assume that the getter method for your logger is named ``getLogger()``, as shown in the previous section. This may differ for you depending on what you named your getter method.
+
+.. code-block:: java
+
+    import java.util.logging.Level;
+    import java.util.logging.Logger;
+
+    getLogger().log(Level level, String message);
+
+The ``level`` is the severity of the log message, and can be any field specified within the ``java.util.logging.Level`` class. The ``message`` is the message you want to be shown. For example:
+
+.. code-block:: java
+
+    getLogger.log(Level.WARNING, "Initialization of this one thing failed!");
```

Comment:
Whoops. Thanks, not sure how I missed that. 

Normally, I would too, though I feel that's something that would be better discussed in another guide because it's applicable to more than loggers.